csuntechlab/takeoff
# TKO-15 FE: Create sign up page & TKO-14 FE: Create login page

What I did:
Created a sign up and a login page for the student

How to test my code:
Pull this branch, see if the sign up and the login page have the basic layout as the mockup. Click on the sign up button and check if it takes you to the login page and vice versa.
